roman-numerals/roman_numerals.c

```diff
@@ -1 +1,43 @@
 #include "roman_numerals.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct{
+    unsigned int base;
+    const char* numeral;
+    const int size;
+} Roman_Numeral_Values_t;
+
+static const Roman_Numeral_Values_t romanNumeralTable[]=
+{
+    { 1000, "M" , 1},
+    { 900, "CM", 2},
+    { 500, "D" , 1},
+    { 400, "CD", 2},
+    { 100, "C" , 1},
+    { 90, "XC" , 2},
+    { 50, "L" , 1},
+    { 40, "XL" , 2},
+    { 10, "X" , 1},
+    { 9, "IX" , 2},
+    { 5, "V" , 1},
+    { 4, "IV" , 2},
+    { 1, "I" , 1}
```

Comment:
can you make it happen with just 2 columns?

Reply:
yeah, I could probably do that